File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/hereDeveloper/discoverExplore/Structured.java

package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Structured {

    @SerializedName("start")
    @Expose
    private String start;
    @SerializedName("duration")
    @Expose
    private String duration;
    @SerializedName("recurrence")
    @Expose
    private String recurrence;

    public String getStart() {
        return start;
    }

    public void setStart(String start) {
        this.start = start;
    }

    public String getDuration() {
        return duration;
    }

    public void setDuration(String duration) {
        this.duration = duration;
    }

    public String getRecurrence() {
        return recurrence;
    }

    public void setRecurrence(String recurrence) {
        this.recurrence = recurrence;
    }

}

File: shared/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/hereDeveloper/discoverExplore/Structured.kt
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore

import com.google.gson.annotations.Expose

class Structured {
    @SerializedName("start")
    @Expose
    private var start: String? = null

    @SerializedName("duration")
    @Expose
    private var duration: String? = null

    @SerializedName("recurrence")
    @Expose
    private var recurrence: String? = null
    fun getStart(): String? {
        return start
    }

    fun setStart(start: String?) {
        this.start = start
    }

    fun getDuration(): String? {
        return duration
    }

    fun setDuration(duration: String?) {
        this.duration = duration
    }

    fun getRecurrence(): String? {
        return recurrence
    }

    fun setRecurrence(recurrence: String?) {
        this.recurrence = recurrence
    }
}